src/proto/semantifly.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+package semantifly;
+
+option go_package = "accretional.com/semantifly/proto";
+
+service Semantifly {
+  rpc Add(AddRequest) returns (AddResponse) {}
+  rpc Delete(DeleteRequest) returns (DeleteResponse) {}
+  rpc Get(GetRequest) returns (GetResponse) {}
+  rpc Update(UpdateRequest) returns (UpdateResponse) {}
+  rpc LexicalSearch(LexicalSearchRequest) returns (LexicalSearchResponse) {}
+}
+
+message AddRequest {
+  string index_path = 1;
```

Comment:
remove index_path from all of these proto messages, this should be supplied at init for the grpc server via command line args. Using it here leaks implementation details to the client and also obviates most of the reason for running semantifly as a server (having the index already loaded into memory when serving eg a Get request)

Reply:
fixed